packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -1633,6 +1633,7 @@ abstract class SqlServerInstanceEngineBase extends InstanceEngineBase {
         s3Import: 'S3_INTEGRATION',
         s3Export: 'S3_INTEGRATION',
       },
+      engineFamily: 'SQLSERVER',
```

Comment:
See the documentation:
```
EngineFamily
The kinds of databases that the proxy can connect to. This value determines which database network protocol the proxy recognizes when it interprets network traffic to and from the database. For Aurora MySQL, RDS for MariaDB, and RDS for MySQL databases, specify MYSQL. For Aurora PostgreSQL and RDS for PostgreSQL databases, specify POSTGRESQL. For RDS for Microsoft SQL Server, specify SQLSERVER.

Valid values: MYSQL | POSTGRESQL | SQLSERVER

Required: Yes

Type: String

Update requires: [Replacement](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-replacement)
```
This shouldn't be set here. This should be user input via props. Firstly, this could be one of three values and secondly, adding this requires a replacement to update. This will impact every user that has this deployed.

Reply:
yeah, I agree with you. Receiving the input via props would be better.